### PR TITLE
fix(main): replace the frozen splash screen with an Animator

### PR DESCRIPTION
**Pull request by caybro**
this runs on the GPU, therefore not prone to main thread being blocked

(UI metrics/details pending Design confirmation)

Fixes: #9524

### What does the PR do

Fixes frozen splash animation on startup

### Affected areas

AppMain,SplashScreen

### Screenshot of functionality (including design for comparison)

- [x] I've checked the design and this PR matches it

[Záznam obrazovky z 2023-02-10 17-24-49.webm](https://user-images.githubusercontent.com/5377645/218143465-89207836-cdeb-4c34-9c3f-821cb10e3482.webm)

### CODE REVIEW

File: ui/main.qml [hunk 1]
```diff
@@ -317,8 +318,20 @@ StatusWindow {
 
     Component {
         id: appLoadingAnimation
-        SplashScreen {
```
Comment (jrainville) [inline ui/main.qml:320]: I guess we can remove the SplashScreen component?

**Review by jrainville — APPROVED** [review]

File: ui/main.qml (hunk 1 above)
Comment (caybro) [reply ui/main.qml:320]: Well.. maybe yes, but then also the `status_splash_*.gif`?

Comment (caybro) [reply ui/main.qml:320]: Hmm, or even better, I'll move this implementation from main.qml into SplashScreen

Comment (caybro) [reply ui/main.qml]: Done

**Review by osmaczko — APPROVED** [review]

**Comment by glitchminer** [comment]
@caybro , sorry, not sure if it's just my machine or not but I wasn't able to launch the windows build https://ci.status.im/job/status-desktop/job/branches/job/windows/job/PR-9525/7/

**Comment by caybro** [comment]
> @caybro , sorry, not sure if it's just my machine or not but I wasn't able to launch the windows build https://ci.status.im/job/status-desktop/job/branches/job/windows/job/PR-9525/7/

I guess you're getting hit by https://github.com/status-im/status-desktop/issues/9502; you'd need a newer build that contains both fixes (not sure if the newest 0.10 rc7 contains this PR'S changes too) :/

**Pull request merged**